stigs/system/ksp-audit-fstab-file-stig-v-230520.yaml

```diff
@@ -0,0 +1,19 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
```

Comment:
Can you attach ref (like [this](https://github.com/yasin-cs-ko-ak/policy-templates/blob/main/stigs/system/ksp-block-stig-psql-v-214082.yaml#L4)) of STIG URL for this Policy

Reply:
Okay